extensions/objects/templates/SanitizedUrlEditor.js

```diff
@@ -42,18 +42,10 @@ oppia.directive('sanitizedUrlEditor', [
         };
       }, true);
 
-      $scope.getWarningText = function() {
-        if ($scope.localValue.label.indexOf('http://') !== 0 &&
-            $scope.localValue.label.indexOf('https://') !== 0) {
-          return 'URLs should start with http:// or https://';
-        } else {
-          return '';
-        }
-      };
-
       $scope.alwaysEditable = true;
 
       $scope.$watch('localValue.label', function(newValue) {
+        $scope.$parent.value = encodeURI(newValue);
         if (newValue.indexOf('http://') === 0 ||
```

Comment:
Yep, sorry I missed that. I meant to remove the check after moving the `encodeURI` out of the conditional.